include/eve/module/math/regular/nthroot.hpp

```diff
@@ -11,80 +11,61 @@
-  //!
-  //! ---
-  //!
-  //!  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~{.cpp}
-  //!  template< floating_real_value T,integral_value U > auto operator()( T x, U n) const noexcept;
-  //!  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-  //!
-  //! **Parameters**
-  //!
-  //!`x`:   [floating real value](@ref eve::floating_real_value).
-  //!
-  //!`n`:   [integral_value](@ref eve::integral_value). Actually `n` can be a [flint](@ref eve::is_flint).
-  //!
-  //! **Return value**
-  //!
-  //!Returns [elementwise](@ref glossary_elementwise) the value of \f$x^{1/n}\f$.
-  //!For negative `x` the value returned is a Nan as soon as `n` is not an odd integer.
-  //!
-  //! The result type is of the compatibility type of the  parameters.
-  //!
-  //! ---
-  //!
-  //!  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~{.cpp}
-  //!  auto operator[]( conditional_expression auto cond ) const noexcept;
-  //!  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-  //!
-  //!  Higher-order function generating a masked version of eve::nthroot
-  //!
-  //!  **Parameters**
-  //!
-  //!  `cond` : conditional expression
-  //!
-  //!  **Return value**
-  //!
-  //!  A Callable object so that the expression `nthroot[cond](x, ...)` is equivalent to `if_else(cond,nthroot(x, ...),x)`
-  //!
-  //! ---
-  //!
-  //! #### Supported decorators
-  //!
-  //!  * eve::raw
-  //!
-  //!
-  //!     The expression `raw(nthroot)(x,n)` does not care about limiting values and gives less accurate values.
-  //!      for instance `nthroot(64.0,3)` is not exactly four but is `3.9999999999999991118` with a 0.5ulp error.
-  //!
-  //!  * eve::diff, eve::diff_1st, eve::diff_nth
-  //!
-  //!
-  //!     The expression `diff_1st(nthroot)(x,n)`
-  //!      diff of \f$f\f$, where \f$f\f$ is the function \f$x \rightarrow \ x^{1/n}\f$.
-  //!
-  //! #### Example
-  //!
-  //! @godbolt{doc/math/nthroot.cpp}
-  //!
-  //!  @}
-  //================================================================================================
-  EVE_MAKE_CALLABLE(nthroot_, nthroot);
+//================================================================================================
+//! @addtogroup math_exp
+//! @{
+//! @var nthroot
+//!
+//! @brief Callable object computing the nth root: \f$x^{1/n}\f$.
+//!
+//!   **Defined in Header**
+//!
+//!   @code
+//!   #include <eve/module/math.hpp>
+//!   @endcode
+//!
+//!   @groupheader{Callable Signatures}
+//!
+//!   @code
+//!   namespace eve
+//!   {
+//!      template< eve::floating_value T, eve::floating_value U >
+//!      T nthroot(T x, U y) noexcept;
+//!   }
+//!   @endcode
+//!
+//! **Parameters**
+//!
+//!   * `x`:   [floating real value](@ref eve::floating_real_value).
+//!
+//!   * `n`:   [integral_value](@ref eve::integral_value). Actually `n` can be a [flint](@ref
+//!   eve::is_flint).
+//!
+//! **Return value**
+//!
+//!    *  Returns [elementwise](@ref glossary_elementwise) the value of \f$x^{1/n}\f$.
```

Comment:
Again, we have a single return, so a single sentence is better

Reply:
done